### DISCUSSION TITLE
请问大家在测评自己量化的qwen模型时，具体该如何操作。文档里“新增第三方模型”教程有点笼统，看不明白。

### DISCUSSION THREAD

**Discussion by AriesHaa** [open-compass/opencompass#1265] [Q&A]
-如题-

**Comment by christyzou** [comment]
你有两种方法可以测评自定义的模型：
方法一：自定义传参，--hf-path换成你自己的模型路径
```
python run.py \
--datasets ceval_gen \   
--hf-path /mnt/diskb5/LLM/model/XXX \  
--tokenizer-path /mnt/diskb5/LLM/model/XXX \
--tokenizer-kwargs padding_side='left' truncation='left' trust_remote_code=True \
--model-kwargs trust_remote_code=True device_map='auto' \
--max-seq-len 2048 \
--max-out-len 32\
--batch-size 4 \
--hf-num-gpus 1 \
--debug
```
方法二：增添配置文件：
第一步：
![image](https://github.com/user-attachments/assets/0f57b5f9-7f31-43cb-8472-60897608c105)
第二步：
![image](https://github.com/user-attachments/assets/b71c9d03-922d-40ea-ab63-9a70c9076c7b)
![image](https://github.com/user-attachments/assets/406a521c-3e65-44f5-91e9-8550865e6a92)
第三步：
![image](https://github.com/user-attachments/assets/e6a8a78f-9f42-4720-b5e9-686d6c991196)
